### ISSUE TITLE
Visual Suggestion

### ISSUE THREAD

**Issue by mclemente** [ironmonk88/monks-tokenbar#36]
Hey, `.monks-tokenbar.chat-card .card-header h3` (line 415 on the .css) could use flex:3 instead of flex:2. Currently, there's a line wrap because of the CD field depending on the size of the line. Here's some prints of the Portuguese translation.
Flex:2
![image](https://user-images.githubusercontent.com/5288872/108790637-f94d5280-755b-11eb-81b2-2bb3460041d3.png)
Flex:3
![image](https://user-images.githubusercontent.com/5288872/108790659-08cc9b80-755c-11eb-9de1-097707046de0.png)

**Comment by mclemente** [comment]
I can just fix that on the system-side by setting up a css, that way I can even have it conform with the system's pattern.

**Comment by mclemente** [comment]
Actually, the problem isn't the CSS, but the HTML.

```html
<div class="item-controls flexrow" style="text-align:right;">
  <h3 class="gm-only">{{localize 'MonksTokenBar.SavingDC'}}: {{dc}}</h3>{{/if}}
</div>
```
Will always create that div, while the following might do the trick.
```html
{{#if dc}}
  <div class="item-controls flexrow" style="text-align:right;">
    <h3 class="gm-only">{{localize 'MonksTokenBar.SavingDC'}}: {{dc}}</h3>
  </div>
{{/if}}
```

**Comment by ironmonk88** [comment]
Good suggestion.  Added that, but also updated the css to limit it's size and to make sure the title stays on one line.

**Comment by ironmonk88** [comment]
DC is now wrapped as you suggest and released in 1.0.17.
